### ISSUE TITLE
Python and C++ snippets are both displayed

### ISSUE THREAD

**Issue by Moguri** [Moguri/panda-sphinx#1]
We probably need to use something like the "ifconfig" extension or something like that to allow building a python and C++ version, or something that will allow us to assign a custom CSS class to toggle Python/C++ sections.

Example page with issue: https://moguri.github.io/panda-sphinx/intro/tutorial/loading-the-grassy-scenery.html

**Comment by tcdude** [comment]
How about using [this](https://sphinxcontrib-contentui.readthedocs.io/en/latest/about.html) extension to resolve this? This way, every sample stub could have a `Python` and `C++` header, that the user can toggle.
See [example](https://sphinxcontrib-contentui.readthedocs.io/en/latest/tabs.html#more-examples)

Would be nice though, if we could have some sort of global link to toggle the preferred one, so that one can choose `C++` once and then always see the C++ sample

**Comment by rdb** [comment]
Yeah, that approach seems feasible.  Just like how Google does it on this page:
https://developers.google.com/identity/protocols/OAuth2UserAgent
It looks like clicking it in one place will switch them everywhere on the page.
